src/libraries/Microsoft.PowerFx.Core/Syntax/Visitors/TexlFunctionalVisitor.cs

```diff
@@ -41,6 +41,14 @@ public abstract class TexlFunctionalVisitor<TResult, TContext>
         /// <param name="context">The context passed to the node.</param>
         /// <returns>The node visit result.</returns>
         public abstract TResult Visit(StrLitNode node, TContext context);
+
+        /// <summary>
+        /// Visit <see cref="TypeLiteralNode"/> leaf node.
+        /// </summary>
+        /// <param name="node">The visited node.</param>
+        /// <param name="context">The context passed to the node.</param>
+        /// <returns>The node visit result.</returns>
+        public abstract TResult Visit(TypeLiteralNode node, TContext context);

```

Comment:
>abstract [](http://example.com/codeflow?start=15&length=8)

these new abstracts will be API breaking. 
Can we  make them virtual (not abstract) to avoid the break?